handlers/grafana_handlers.go

```diff
@@ -19,6 +19,20 @@ func init() {
 	gob.Register(&models.GrafanaClient{})
 }
 
+// swagger:route POST /api/telemetry/metrics/grafana/config GrafanaAPI idPostGrafanaConfig
+// Handle POST request for Grafana configuration
+//
+// Used for persisting Grafana configuration
+// responses:
+// 	200:
+
+// swagger:route DELETE /api/telemetry/metrics/grafana/config GrafanaAPI idDeleteGrafanaConfig
+// Handle DELETE request for Grafana configuration
+//
+// Used for Delete Grafana configuration
+// responses:
+// 	200:
+
```

Comment:
Can you add a `GET` request for:
`/api/telemetry/metrics/grafana/config` and `/api/telemetry/metrics/prometheus/config`.
It was added in